indra/tools/disambiguate.py

```diff
@@ -0,0 +1,115 @@
+import time
+import logging
+
+from indra.literature.elsevier_client import logger as elsevier_logger
```

Comment:
Instead of importing the logger object, it's better to refer to the module in the logger hierarchy as:
```
el = logging.getLogger('indra.literature.elsevier_client')
el.setLevel(logging.WARNING)
```